libs/ballot-interpreter-nh-next/src/election.rs

```diff
@@ -16,6 +16,14 @@ idtype!(PrecinctId);
 //
 // IF YOU CHANGE ANYTHING HERE, YOU MUST ALSO CHANGE IT THERE.
 
+#[derive(Debug, Clone, Serialize, Deserialize)]
+pub struct Rect {
+    pub x: i32,
+    pub y: i32,
+    pub width: u32,
+    pub height: u32,
+}
```

Comment:
It might be nice to unify `geometry::Rect` with `election::Rect`

Reply:
Ended up creating an `Outset` struct instead.

Reply:
Nice, that's more descriptive